system-tests/package.json

```diff
@@ -68,6 +68,7 @@
     "multer": "1.4.2",
     "nock": "12.0.2",
     "proxyquire": "2.1.3",
+    "react-scripts": "5.0.0",
```

Comment:
Why do we need this here? System test projects can install their own dependencies

Reply:
Yes, but in the test for CT, we assume cypress did not discover it.
The tested scenario is the monorepo where CRA is installed at the top.

Once we go to the last step, we still need the initialization to happen properly.